### ISSUE TITLE
HumanPhyCtrl

### ISSUE THREAD

**Issue by exmex** [patheagames2019/planetexplorers#2]
Class HumanPhyCtrl is missing. Where do I get that from?

Also 'TestPEEntityCamCtrl' missing.

**Comment by JeremyARussell** [comment]
I would like to second the request for some more information on how we can get these plugins.

@exmex  I noticed that you have created those files on your fork, did you have any luck getting things running after doing that?

**Comment by exmex** [comment]
Yes. With all plugins and those two files added to my fork it works.

**Comment by JeremyARussell** [comment]
@exmex Oh awesome, what plugins did you end up needing and where can you find them? I was unable to track down the uLink plugin and another I tracked down doesn't seem to have a free version.

**Comment by exmex** [comment]
Google ragezone, there you can get the files with plugins
